cmd/controller/main.go

```diff
@@ -119,22 +109,19 @@ func main() {
 		"interval at which the controller will re list resources even with no changes, in seconds.")
 	flag.IntVar(&queueMaxRetries, "dynamic-controller-default-queue-max-retries", 20,
 		"maximum number of retries for an item in the queue will be retried before being dropped")
-	// log level flags
-	flag.IntVar(&logLevel, "log-level", 10, "The log level verbosity. 0 is the least verbose, 5 is the most verbose.")
 	// qps and burst
 	flag.Float64Var(&qps, "client-qps", 100, "The number of queries per second to allow")
 	flag.IntVar(&burst, "client-burst", 150,
 		"The number of requests that can be stored for processing before the server starts enforcing the QPS limit")
 
-	flag.Parse()
-
 	opts := zap.Options{
 		Development: true,
```

Comment:
Just to follow-up on #828, even if we have `Development: true` `-zap-encoder=json` will enable json logs

<img width="1120" height="157" alt="Image" src="https://github.com/user-attachments/assets/936887cc-7b7a-4bac-9206-de1eba7f3866" />